types/configs/conversion-config.ts

```diff
@@ -12,7 +12,7 @@ export interface ConversionConfig {
    * 1. String — the key of Tool data object to fill it with imported string on render.
    * 2. Function — method that accepts importing string and composes Tool data to render.
    */
-  import?: ((data: string) => string) | string;
+  import?: ((data: string, config: ToolConfig) => BlockToolData) | string;
```

Comment:
import can be both a string and a function that is returning a string

"string" means a block data property that should be filled.

For example for Header tool:

<img width="663" alt="image" src="https://github.com/user-attachments/assets/6f6a16cd-d225-4dd4-8eed-fb1bba3906d1">

Reply:
sorry, you're right, that import should return a `BlockToolData`